Cargo.toml

```diff
@@ -83,6 +84,8 @@ puzzles = ["dep:chia-puzzles"]
 clvm-traits = ["dep:clvm-traits"]
 clvm-utils = ["dep:clvm-utils"]
 
+openssl = ["clvmr/openssl"] # enable openssl for clvmr on unix
```

Comment:
This will be decided by dependents and/or the CI here, not this manifest. So this comment is unnecessary imo.

Reply:
removed